### PR TITLE
Optimze ACL Table/Rule notification handling

### PR DESCRIPTION
**Pull request by abdosi**
What I did:
Optimize ACL Table/Rule notification handling
to loop pop() until empty to consume all the data in a batch
This way we prevent multiple call to iptable updates

**- How to verify it**

Loaded (sudo acl-loader update full acl.json) with 100 rules and verified all the set notification are batch together in one update to iptables .

Similarly (sudo acl-loader delete) batch all del operation in one update.

### CODE REVIEW

**Review by jleveque — APPROVED** [review]
Looks good to me. @qiluo-msft to review, as well.

File: files/image_config/caclmgrd/caclmgrd [hunk 1]
```diff
@@ -550,23 +550,24 @@ class ControlPlaneAclManager(daemon_base.DaemonBase):
...
+                        break
+                    # ACL Table notification. We will take Control Plane ACTION for any ACL Table Event
+                    # This can be optimize further but we should not have many acl table set/del events in normal
+                    # scenario
+                    if acl_rule_table_seprator not in key:
+                        ctrl_plane_acl_notification.add(namespace)
+                    # Check ACL Rule notification and make sure Rule point to ACL Table which is Controlplane
+                    else:
+                        acl_table = key.split(acl_rule_table_seprator)[0]
+                        if self.config_db_map[namespace].get_table(self.ACL_TABLE)[acl_table]["type"]  == self.ACL_TABLE_TYPE_CTRLPLANE:
```
Comment (qiluo-msft) [inline files/image_config/caclmgrd/caclmgrd:566]: >  [](start = 102, length = 1)

Use only one space

Comment (abdosi) [reply files/image_config/caclmgrd/caclmgrd]: @ Done

---
In reply to: [504353871](https://github.com/Azure/sonic-buildimage/pull/5621#discussion_r504353871) [](ancestors = 504353871)

**Review by jleveque — APPROVED** [review]

**Review by qiluo-msft — APPROVED** [review]

**Comment by qiluo-msft** [comment]
Do you have performance test result?

**Comment by abdosi** [comment]
> Do you have performance test result?

@qiluo-msft 
I verified the number of update happening to iptable (function update_control_plane_acls) for 100 rule is only 1 time. Previously it was called 100 times.

I did not measure in time perspective.

**Pull request merged**